src/collection-view.js

```diff
@@ -128,12 +128,33 @@ Marionette.CollectionView = Marionette.View.extend({
     return this;
   },
 
+  // Reorder DOM after sorting. When your element's rendering
+  // do not use their index, you can pass reorderOnSort: true
+  // to only reorder the DOM after a sort instead of rendering
+  // all the collectionView
+  reorder: function () {
+    var children = this.children;
+
+    // get the DOM nodes in the same order as the models
+    var els = this.collection.models.map(function (model) {
```

Comment:
I guess it would have to be `.map(this.children.findByModel, this.children)` and `.pick('el')` ;-) Anyway, I think until `underscore` becomes lazy, the chaining is uglified by the `.value()` at the end (that I always forgot). Also, I try to avoid `_`'s chaining because it is too easy to shoot yourself in the foot with the `this`.